server/data/apps.json

```diff
@@ -1,4 +1,21 @@
 [
+  {
+    "appName": "Audicy",
+    "appType": [
+      "Discord Audio Streaming Bot"
+    ],
+    "appUrl": "https://audicy.xyz",
+    "appIconUrl": "audicylogo.png",
+    "platforms": [
+      "Discord"
```

Comment:
I think `Discord` would need to be added to the list of platforms here: https://github.com/Podcastindex-org/web-ui/blob/45cce47335def6226640f539e5ac3a2b45816a2d/server/data/apps-schema.json#L55-L81